src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useCallback, useState, ChangeEvent, useMemo } from "react";
 import classNames from "classnames";
-import { isEqual } from "lodash";
+import { isEqual, reduce } from "lodash";
```

Comment:
There's no need to use `reduce` from `lodash`. The language already has built in functionality for this: `theArray.reduce`. This needs to be reverted and fixed in a followup PR.

Reply:
I guess that the intention to use `lodash` was that it's faster, at least it's what I found using benchmarks
though not sure it's the idea that @denisdimitrov had originally